articles/machine-learning/how-to-debug-parallel-run-step.md

```diff
@@ -112,6 +112,15 @@ file_path = os.path.join(script_dir, "<file_name>")
 
 You can specify `mini_batch_size`, `node_count`, `process_count_per_node`, `logging_level`, `run_invocation_timeout`, and `run_max_try` as `PipelineParameter`, so that when you resubmit a pipeline run, you can fine-tune the parameter values. In this example, you use `PipelineParameter` for `mini_batch_size` and `Process_count_per_node` and you will change these values when you resubmit another run. 
 
+#### CUDA Devices Visibility
+For compute target with GPU, the environment variable `CUDA_VISIBLE_DEVICES` will be set in worker processes. In AmlCompute, the total number of GPU devices is in environment variable `AZ_BATCHAI_GPU_COUNT_FOUND`, which will be set automatically. If users want each worker process has a dedicated GPU, set `process_count_per_node` same to the number of GPU devices on a machine. Each worker process will assign a unique index to `CUDA_VISIBLE_DEVICES`. If a worker process stops for any reason, the next started worker process will use the released GPU index.
```

Comment:
For compute targets equipped with GPUs, the environment variable `CUDA_VISIBLE_DEVICES` will be set in worker processes. In AmlCompute, you can find the total number of GPU devices in environment variable `AZ_BATCHAI_GPU_COUNT_FOUND`, which is set automatically. If you want each worker process to have a dedicated GPU, set `process_count_per_node` equal to the number of GPU devices on a machine. Each worker process will assign a unique index to `CUDA_VISIBLE_DEVICES`. If a worker process stops for any reason, the next started worker process will use the released GPU index.

Reply:
Thanks for your review. It is improved a lot :)